paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentOptionScreenshotTest.kt

```diff
@@ -0,0 +1,101 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.ui.unit.dp
+import com.stripe.android.paymentsheet.PaymentOptionUi
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.utils.screenshots.FontSize
+import com.stripe.android.utils.screenshots.PaparazziRule
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import com.stripe.android.utils.screenshots.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+class PaymentOptionScreenshotTest {
```

Comment:
Why are these all full screen? We should be able to shrink them to be the width we expect, right?

Reply:
The `boxModifier` of `PaparazziRule` is set to expand to the full width, which I’m fine with. Can change if you want me to, though.

Reply:
I vote yes, since we're testing an item that isn't full width, but I won't block on it.

Reply:
As discussed over Slack, I’ll make a separate change in `PaparazziRule` directly, but merge this as-is.